vars/main.yml

```diff
@@ -1,5 +1,13 @@
 ---
 # vars file for backups_role
+
+# Check if cron script template of prepare backup, is the default one (true) or has been overriden (false)
+_template_is_default: "{{ backups_role_script_prepare_template == 'cron-prepare.sh.j2' }}"
+# If swithces have been set at the inventory, respect them.
+# Otherwise, switch them off when the template has changed.
+_backups_role_sudoers_enabled: "{{ backups_role_sudoers_enabled | default(_template_is_default) }}"
+_backups_role_postgresql_enabled: "{{ backups_role_postgresql_enabled | default(_template_is_default) }}"
+
```

Comment:
I think this could seem a bit of black magic. Why don't we just throw a warning or an error when we detect that the default template is not being used?

Reply:
We don't want to throw an error when the default template is not being used. We want it to be overridable, that's ok. But at the same time, we need this feature not to enforce postgres to be installed and <insert reasoning of the MR description>.

An alternative to this "black magic" is to just give the flags and enforce to user of the role to set them to false (don't run postgres) if they use another script template to prepare the backups. But in my opinion, the default behavior should be a bit smarter than that.

Reply:
I think that including postgres tasks by default is a bit arbitrary since it just comes from what we started to backup.

I also would decouple `sudoers` and `postgresql` tasks, they shouldn't share the same meaning of "default".

I'm starting to think that this role should do nothing by default and that the user should enable the things that she needs to backup. What do you think?

Reply:
I agree. I don't like the idea of assuming what users want to backup. By now I would just merge it, as we need it for opencell, but I would give it more time to think about it. Maybe placing one taskfile for postgres and one extra for tar? And then from the importer playbook "import tasks from" those taskfiles...

Reply:
OK, let's create a Trello card and an issue here in the repo and maybe a comment in the code. I think we should address this issue ASAP. Thanks!